src/plugins/dashboard/server/saved_objects/dashboard_migrations.ts

```diff
@@ -257,14 +307,8 @@ export const createDashboardSavedObjectTypeMigrations = (
     '7.0.0': flow(migrations700),
     '7.3.0': flow(migrations730),
     '7.9.3': flow(migrateMatchAllQuery),
+    '7.10.0': flow(migrateExplicitlyHiddenTitles),
     '7.11.0': flow(createExtractPanelReferencesMigration(deps)),
-
```

Comment:
Removing this comment as it is no longer relevant due to the work @ppisljar did in https://github.com/elastic/kibana/pull/103680